src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_help.py

```diff
@@ -13,3 +13,15 @@
     type: group
     short-summary: Commands to manage a DCOS orchestrated Azure container service.
 """
+helps['acs kubernetes'] = """
+    type: group
+    short-summary: Commands to manage a kubernetes orchestrated Azure container service.
+"""
+helps['acs scale'] = """
+    type: command
+    short-summary: Change private agent count of a container service.
+"""
+helps['acs install-cli'] = """
+    type: command
+    short-summary: Downloads the dcos/kubernetes command line from Mesosphere. Default is dcos.
```

Comment:
The kubernetes command line is not downloaded from Mesosphere, only the dcos one. Maybe remove 'from Mesosphere'?
Also, the default isn't always DCOS. It depends on the orchestrator type of the cluster so I'd remove the default message.